plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -78,8 +78,12 @@ def verification_templates(main_only: false)
       templates.each_with_index.map { |c, i| Kubernetes::TemplateFiller.new(self, c, index: i) }
     end
 
+    def blue_green?
+      kubernetes_role.blue_green?
+    end
+
     def blue_green_color
-      kubernetes_release.blue_green_color if kubernetes_role.blue_green?
+      kubernetes_release.blue_green_color if blue_green?
```

Comment:
would remove the check here so we can use it in the configmap hackery, but good either way

Reply:
I think the configmap will be created within the release_doc similar to the PDB, so we can reference the release color directly. Or I'll remove the check in the configmap PR if necessary?